code-for-nashville/nashville-zone-lookup
# [Issue:79] Added collapsible categories -@jsheridanwells

Issue #79 

Uses are hidden under collapsible categories. 

#### Manual Testing: 

* Search for an address with __Use Category__ as Everything. All categories will appear collapsed. Click any of the categories, and uses for that category will be displayed. 

* Seach for an address with a __Use Category__ selected. Results for that category will be displayed with the category open. Click 'View Other Uses' and other categories for that zone will appear collapsed. 

* When clicking a closed category, the down icon will become an up icon and the background color will change from gray to light blue and vice versa when closing the category. 

* More than one category can be open at the same time.
